src/schema/users.ts

```diff
@@ -850,14 +852,56 @@ export const resolvers: IResolvers<any, Context> = {
         .orderBy('date')
         .getRawMany();
     },
-    userStreak: (_, __, ctx: Context, info): Promise<GQLUserStreak> =>
-      graphorm.queryOneOrFail<GQLUserStreak>(ctx, info, (builder) => ({
-        queryBuilder: builder.queryBuilder.where(
-          `"${builder.alias}"."userId" = :id`,
-          { id: ctx.userId },
-        ),
-        ...builder,
-      })),
+    userStreak: async (_, __, ctx: Context, info): Promise<GQLUserStreak> => {
+      const streak = await ctx.con
+        .getRepository(UserStreak)
+        .createQueryBuilder()
+        .select('"lastViewAt"') // needs to consider user's timezone
+        .addSelect('"userId"')
```

Comment:
Cant we just do this?
```suggestion
        .select('lastViewAt, userId')
```

Reply:
I intentionally separated them. Once we get apply all the timezone fetch and conversion on the `lastViewAt`, it is going to be a mess to look at, so to easily see what are the other fields selected, I moved them to the next line.